Add missing example for querying permissions of a Service Account (#30008)
kubernetes/website: content/en/docs/reference/access-authn-authz/authorization.md

no
```

Also, you can use this command to determine if a Service Account can perform a certain action.

> To check if a Service Account named `dev-sa` in Namespace `dev` can list Pods in the Namespace `dev`,

Reviewer: This could probably be more concise, as the impersonation docs linked above touch on usage w/ service accounts already.

Additionally, I'd consider using two different namespaces in the example.

```suggestion
Similarly, to check whether a Service Account named `dev-sa` in Namespace `dev` can list Pods in the Namespace `target`:

Author: @sejr +1. Thus, committing the change.

```bash

> kubectl auth can-i list pods --namespace dev --as=system:serviceaccount:dev:dev-sa

Reviewer: ```suggestion
kubectl auth can-i list pods \
	--namespace target \
	--as system:serviceaccount:dev:dev-sa
```

no
```

> Similarly, to check whether a Service Account named `dev-sa` in Namespace `dev`

Reviewer 2: ```suggestion
Similarly, to check whether a ServiceAccount named `dev-sa` in Namespace `dev`
```
(it's `kind: ServiceAccount` not `kind: Service Account`)
